homeassistant/components/mqtt/client.py

```diff
@@ -815,8 +814,8 @@ def _exception_message(
             call_back_name = getattr(msg_callback, "__name__")
         return (
             f"Exception in {call_back_name} when handling msg on "
-            f"'{msg.topic}': '{msg.payload}'"  # type: ignore[str-bytes-safe]
-        )
+            f"'{msg.topic}': '{msg.payload}'"
```

Comment:
@jbouwh Can you confirm what the desired behaviour is here what mypy is complaining about?

Reply:
Oh never mind, ruff moved the type ignore to the next line